game_catalog/custom_permissions.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class IsAdminOrReadOnly(BasePermission):
+    def has_permission(self, request, view):
+        if request.method in SAFE_METHODS:
+            return True
+        else:
+            return request.user.is_staff
```

Comment:
In Django it seems like good options, but I prefer ternar here.
```suggestion
    def has_permission(self, request, view):
        return True if request.method in SAFE_METHODS else request.user.is_staff
```